_config.yml

```diff
@@ -26,8 +28,8 @@ permalink:        pretty
 
 # Setup
 title:            The OpenTracing Project
-tagline:          'Common APIs for distributed tracing and context propagation'
-description:      'Common APIs for distributed tracing and context propagation'
+tagline:          'Safe, general APIs for distributed tracing and context propagation'
+description:      'Safe, general APIs for distributed tracing and context propagation'
```

Comment:
"safe" in what sense?

Would be nice the first word to emphasize the interop property of the API.